denops/ddc-source-nvim-lsp/completion_item.ts

```diff
@@ -185,9 +185,10 @@ export class CompletionItem {
   toDdcItem(
     lspItem: LSP.CompletionItem,
     defaults?: LSP.CompletionList["itemDefaults"],
+    snippetIndicator: string,
```

Comment:
```suggestion
    snippetIndicator: string,
    defaults?: LSP.CompletionList["itemDefaults"],
```

The optional parameter must be the last.